src/lib/data/npm.ts

```diff
@@ -59,4 +61,24 @@ export class NpmCollector {
 
     return await response.json() as NpmDownloadData;
   }
+
+  async fetchAuthorPackageCount(): Promise<number | undefined> {
+    // Use the first maintainer to search for their packages (libraries can have multiple maintainers)
+    const maintainer = this.packageData?.maintainers?.[0];
+    if (!maintainer?.name) {
+      return undefined;
+    }
+
+    try {
+      const response = await fetch(`https://registry.npmjs.org/-/v1/search?text=maintainer:${encodeURIComponent(maintainer.name)}&size=1`);
```

Comment:
this (and your other fetch calls) are ugly. in a separate PR, see if you can refactor and "build" these endpoints in a helper function so you don't have string concatenation as part of the main `fetchAuthorPackageCount` function